src/browser.js

```diff
@@ -50,6 +50,11 @@ class UniversalEmitter extends Emitter {
   from(): UniversalEmitter {
     return this;
   }
+  flushBeforeTerminated = () => {
+    if (document.hidden) {
+      this.flushInternal();
```

Comment:
Why `this.flushInternal()` vs `this.flush()`?

Reply:
`bind` is slow and we already have the this context